### PR TITLE
f-footer@v8.8.0 Update supported country list & change redirection from twitter to X platform

### PR DESCRIPTION
**Pull request by phatpt8**
__*This is the standard PR description template.*__

__*Delete the parts of this template that aren’t relevant to the current PR you are submitting – for example, you may not need to amend documentation or testing of the component.*__

---

_A short description of your PR goes here. Feel free to copy & paste the CHANGELOG entry that you've added here._

- Stop Portugal, Romania, and New Zealand support 
- Support new country Slovakia
- Redirect user from Twitter to X
    - still keep the Twitter icon because it is not available in the Icon package
    - Keep the same GTM tag so as not to affect the tracking data.

---

## UI Review Checks

image

- [ ] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [ ] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/organisms/f-footer/src/tenants/en-AU.js [hunk 1]
```diff
@@ -71,9 +71,9 @@ export default {
             gtm: 'click_follow_instagram'
         },
         {
-            url: 'https://twitter.com/menulog',
-            name: 'twitter',
-            alt: 'Menu Log on Twitter (leads to external site)',
+            url: 'https://www.x.com/menulog',
+            name: 'X',
```
Comment (xander-marjoram) [inline packages/components/organisms/f-footer/src/tenants/en-AU.js]: Nit: I think it's probably more reliable to use lowercase for the `name`

File: packages/components/organisms/f-footer/src/components/BaseProviderIcon.vue [hunk 2]
```diff
@@ -20,7 +20,8 @@ import {
 // Social icons
 import {
     SocialFacebookIcon as FacebookIcon,
-    SocialTwitterIcon as TwitterIcon,
+    // TODO: Replace with correct icon for X platform when available
```
Comment (jamieomaguire) [inline packages/components/organisms/f-footer/src/components/BaseProviderIcon.vue]: @phatpt8 this icon should be available already
Screenshot 2024-05-21 at 16 07 49

https://pie.design/foundations/iconography/library/

Comment (phatpt8) [reply packages/components/organisms/f-footer/src/components/BaseProviderIcon.vue]: Resolved offline. 
The current "@justeattakeaway/pie-icons-vue" package's version is "2.0.0-beta.1", not including the new Twitter icons.

Comment (phatpt8) [reply packages/components/organisms/f-footer/src/components/BaseProviderIcon.vue]: I updated the TODO comment instead.

**Review by xander-marjoram — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**